README.md

```diff
@@ -137,34 +139,54 @@ The dependency keys can be any strings, but it's recommended to use package name
 
 Examples:
 
-```js
+```json
 // Clone 'git@github.com:cksource/foo.git' and check out to 'master'.
 {
     "foo": "git@github.com:cksource/foo.git"
 }
 ```
 
-```js
+```json
 // Short format. Clone 'git@github.com:cksource/foo.git' and check out to branch 'dev'.
 {
     "@cksource/foo": "cksource/foo#dev"
 }
 ```
 
-```js
+```json
 // Clone 'https://github.com/cksource/foo.git' (via HTTPS) and check out to tag 'v1.2.3'.
 {
     "foo": "https://github.com/cksource/foo.git@v1.2.3"
 }
 ```
 
-```js
+```json
 // Clone 'cksource/foo' and check out to the latest available tag.
 {
     "foo": "cksource/foo@latest"
 }
 ```
 
+### The `presets` option
+
+This option allows the user to create an easy switch between different states of the dependencies. When using any command with the `--preset` option, it will behave as if the `dependencies` option was using values from the given preset. Dependencies that are not defined in the preset but are defined in the `dependencies`, will use version from the `dependencies` as a fallback.
```

Comment:
```suggestion
This option allows the user to switch between different states of dependencies easily. When using any command with the `--preset` option, it will behave as if the `dependencies` option was using values from the given preset. Dependencies not specified in the preset but in the `dependencies` object will use a version from the latter as a fallback.
```